package.json

```diff
@@ -32,6 +32,7 @@
   },
   "dependencies": {
     "core-js": "^2.6.6",
+    "underscore": "^1.9.1",
     "vorpal": "^1.11.4"
```

Comment:
Good work taking the initiative to use the `underscore` library. For school projects it might be wise to ask first for things like this, just in case struggling with this sort of functionality is part of the point of the project (it wasn't), but in an industry setting this would definitely be the right call.